### DISCUSSION TITLE
Feature or enhancement or bug?  Don't login to account when disabling an alias from email thread

### DISCUSSION THREAD

**Discussion by luigicalamar** [anonaddy/anonaddy#251] [Feature Requests]
Not sure if this is intentional or required?

When I select 'here' to disable an alias from within my email (see below), a login window opens up in my browser that logs me into my anonaddy account?  It seems the alias is disabled already - why do I need to have the account opened?

This email was sent to alias@customdomain.com from [spammer@spamdomain.com](url) and has been forwarded by AnonAddy
Click here to deactivate this alias

**Comment by willbrowningme** [comment]
This is intentional, you must be logged in so that you can deactivate the alias. This is to prevent anyone else being able to deactivate any of your aliases.

The link redirects you back to your dashboard once it has deactivated the alias in question.

**Reply by terrabitz** [reply]
I know this is an old thread, but I would like an option to disable auth on these links as well. I know it slightly decreases security, but it would help in my use case.

I manage a couple of domains that get forwarded to the email addresses of my (non-technical) family members. If they want to block an alias, they either have to set up a rule in their email client, or they need my credentials to AnonAddy.

In general, I'm not terribly worried about the loss of security since it looks like the links use a random GUID that someone would need to brute force. In theory someone that intercepts these messages or gets them forwarded to them could also click these links, but IMO the risk isn't too big of a deal.

One other potential issue is any middleboxes automatically investigating links in the email (e.g. for security scanning). Then you'd need a mechanism to prevent these from accidentally disabling aliases. If that's an issue, it could be solved with a confirmation button on the resulting web page.
